src/lm/arpa-file-parser.h

```diff
@@ -43,13 +44,17 @@ struct ArpaParseOptions {
 
   ArpaParseOptions()
       : bos_symbol(-1), eos_symbol(-1), unk_symbol(-1),
-        oov_handling(kRaiseError), use_log10(false) { }
+        oov_handling(kRaiseError), use_log10(false),
```

Comment:
I'll make a separate PR. It will affect *a lot* of tests. I used it to simplify testing (read -1.23, expect -1.23 in the NGram structure). It seems a small change, but it is big in the number of lines, and I do not want to mix the to, to keep them reviewable.